src/app/app.component.ts

```diff
@@ -18,6 +20,7 @@ export class AppComponent implements OnInit{
 
   isWindowMobile(event: any): void {
     this.MobileView = event.innerWidth <= 750;
+    this.store.dispatch(new ToggleMobileScreen(this.MobileView))
```

Comment:
I am not sure this functionality should be implemented in the app component. I believe it should be moved to header or result components.

Reply:
it was decided to move it from header to app component to have the possibility to pass isMobile flag to multiple child components without creating additional subscriptions